File: library/src/main/java/com/harvey/arcface/utils/ILogger.java
package com.harvey.arcface.utils;

public interface ILogger {

    void showLog(boolean isShowLog);

    void showStackTrace(boolean isShowStackTrace);

    void d(String message);

    void i(String message);

    void w(String message);

    void e(String message);

    void monitor(String message);

    boolean isMonitorMode();

    String getDefaultTag();
}

File: kotlin-library/src/main/java/com/harvey/arcface/utils/ILogger.kt
package com.harvey.arcface.utils

interface ILogger {

    val isMonitorMode: Boolean

    val defaultTag: String

    fun showLog(isShowLog: Boolean)

    fun showStackTrace(isShowStackTrace: Boolean)

    fun d(message: String)

    fun i(message: String)

    fun w(message: String)

    fun e(message: String)

    fun monitor(message: String)
}
